code/gui/mainwindow.ui

```diff
@@ -123,6 +153,56 @@ p, li { white-space: pre-wrap; }
      </item>
     </layout>
    </widget>
+   <widget class="QWidget" name="layoutWidget_2">
+    <property name="geometry">
+     <rect>
+      <x>40</x>
+      <y>390</y>
+      <width>571</width>
+      <height>69</height>
+     </rect>
+    </property>
+    <layout class="QVBoxLayout" name="controller_interface">
+     <item>
+      <widget class="QLabel" name="controller_label">
+       <property name="text">
+        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Select controller&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <layout class="QHBoxLayout" name="controller_layout">
+       <item>
+        <widget class="QComboBox" name="controller_type">
+         <item>
+          <property name="text">
+           <string>Actuator</string>
+          </property>
+         </item>
+         <item>
+          <property name="text">
+           <string>Simulator</string>
+          </property>
+         </item>
+        </widget>
+       </item>
+       <item>
+        <widget class="QPushButton" name="controller_button">
+         <property name="minimumSize">
+          <size>
+           <width>125</width>
+           <height>20</height>
+          </size>
+         </property>
+         <property name="text">
+          <string>Select</string>
+         </property>
+        </widget>
+       </item>
+      </layout>
+     </item>
+    </layout>
+   </widget>

```

Comment:
The option to switch controller needs to be in the menu, Setup -> Simulator Mode. Right now, you will see two greyed out options: `EMMA Actuator mode` and `EMMA Solenoid mode`, replace them with `SAM mode` and `Simulator mode` and enable those radio options.